scripts/trivydb2tc.py

```diff
@@ -468,104 +427,103 @@ def main() -> None:
     with bdb.view() as txs:
         for repos, _ in txs.bucket():
             if repos in allow_list:
-                # create tag strings except packagename
                 category = get_package_info(repos)
-                vulns = vuln_info(repos, txs)
-                for vuln in vulns:
-                    solution, vuln_vers, fix_vers = solution_from_vuln(vuln)
-                    if not solution:
+                trivy_vulns = vuln_info(repos, txs)
+                for vuln in trivy_vulns:
+                    vuln_vers, fix_vers = get_versions_from_trivy_vuln(vuln)
+                    if vuln_vers is None and fix_vers is None:
                         continue
-                    vuln_id = vuln["vuln_id"]
-                    vuln_obj = vuln_dict.get(
-                        vuln_id, {"tags": set(), "actions": {}, "fix_vers": {}}
-                    )
-                    tag = f"{vuln['pkg_name']}:{category}"
-                    assert isinstance(vuln_obj["tags"], set)
-                    vuln_obj["tags"].add(tag)
-                    assert isinstance(vuln_obj["actions"], dict)
-                    act_obj = vuln_obj["actions"].get(
-                        solution,
+                    trivy_vuln_id = vuln["vuln_id"]
+                    vuln_obj = vuln_dict.get(trivy_vuln_id, {"affects": {}})
+                    package = f"{vuln['pkg_name']}:{category}"
```

Comment:
package_name と category を文字列型で接続して、後工程で分離するのはパースのミスなどの可能性が出ると思います。
ここではpackageはdictのkeyとしての利用なので、tuple にするなどの方が安全に思います
```suggestion
                    package = (vuln['pkg_name'], category)
```